src/AcceptanceTests/App_Packages/NSB.AcceptanceTests.6.0.0-unstable1607/Routing/When_publishing_to_scaled_out_subscribers_on_multicast_transports.cs

```diff
@@ -3,13 +3,13 @@
     using System.Threading;
     using System.Threading.Tasks;
     using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
     using NServiceBus.AcceptanceTests.ScenarioDescriptors;
     using NUnit.Framework;
 
     public class When_publishing_to_scaled_out_subscribers_on_multicast_transports : NServiceBusAcceptanceTest
     {
-        // TODO: address when core is update
-        [Test, Explicit("Requires a fix in core before can be executed")]
+        [Test]
```

Comment:
Not sure why this got removed. Until instance-id issue is resolved with core, this test won't run.